lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import Tab from './Tab';
+
 const Tabs = props => {
-  return (
-    <div className="tabs">
-      <div className="topics">
-        <span className="title">TRENDING TOPICS:</span>
-        {/* map over the tabs provided on your props, create a new Tab component for each one.
-            give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+  console.log(props)
```

Comment:
Nice job loggin the props out to the console for debugging and trouble shooting.